graylog2-web-interface/devServer.js

```diff
@@ -37,8 +38,14 @@ const appConfig = webpackConfig[1];
 const vendorCompiler = webpack(vendorConfig);
 const appCompiler = webpack(appConfig);
 
+const { argv } = yargs;
+const host = argv.host || DEFAULT_HOST;
+const port = argv.port || DEFAULT_PORT;
+const apiUrl = argv.apiUrl || process.env.GRAYLOG_API_URL || DEFAULT_API_URL;
+
 // Proxy all "/api" requests to the server backend API.
-app.use('/api', proxy('localhost:9000', {
+console.log(`Graylog web interface forwarding /api requests to ${apiUrl}`);
```

Comment:
hmm, eslint is upset about this. should we add a rule to disable tne `no-consle` check?

Reply:
Oups, good catch. We already have it for the other console log statements. I will add it.